Formula/paket.rb

```diff
@@ -0,0 +1,46 @@
+class Paket < Formula
+  desc "Dependency manager for .NET with support for NuGet and Git repositories"
+  homepage "https://fsprojects.github.io/Paket/"
+  url "https://github.com/fsprojects/Paket/releases/download/3.7.3/paket.exe"
+  sha256 "fcb4efb71c78831b89b15c39a793165d34172677363d66e005e6789afb76fc2c"
+
+  bottle :unneeded
+
+  depends_on "mono"
```

Comment:
Mind making this:
```ruby
depends_on "mono" => :recommended
```
We tend to stick to that as there are people who prefer using the upstream Mono package, so that gives them a way to opt-out.